pages/scrollytelling.vue

```diff
@@ -162,41 +163,27 @@ const stepIndex = ref(0); // keep track of the index
 const stepProgress = ref(0); // keep track of the progress within a step
 // const pageProgress = ref(0) // keep track of the total page progress
 
-// const grainType = useActiveGrainType()
-const grainType = ref(
```

Comment:
This ref is the source of truth for the active grain type, as it will be changed by the scrolling

Reply:
could it use `setActiveGrainType` from the composable? i thought the idea was to avoid having refs in components, just pulling from composables

Reply:
Yeah but can you just move the string to here instead of in the composable, so use setActiveGrainType in this spot instead? Off the ref?